tests/integration/src/test/java/org/mule/test/integration/exceptions/ExceptionHandlingTestCase.java

```diff
@@ -132,7 +131,9 @@ public void testOutboundEndpointInScope() throws Exception
         MuleMessage response = client.request("test://outScope2", 3000);
 
         assertNotNull(response);
-        assertTrue(response.getProperty("expectedHandler", PropertyScope.SESSION));
+
+        FlowAssert.verify("outboundEndpointInScope");
```

Comment:
Can't static import FlowAssert.verify and Mockito.verrify.  In this cases where both are used it's best not to static import either and be explicit.